translate/src/modules/comments/components/AddComment.tsx

```diff
@@ -108,10 +108,10 @@ export function AddComment({
         children: [{ text: name }],
       };
       Transforms.insertNodes(editor, mention);
-      Transforms.move(editor);
+      Transforms.select(editor, Editor.end(editor, []));
       Transforms.insertText(editor, ' ');
     },
-    [],
+    [editor],
```

Comment:
This addition is fine, but unnecessary in this case as the value is coming from `useMemo`.

Reply:
So let's remove that? 

It doesn't seem to fix the problem where you add a mention in the middle of the comment - when you insert it, the space is added to the end of the string and the cursor jumps there, too.

Reply:
Including the `editor` as a dependency is nominally good practice; skipping it is such a marginal optimization that it doesn't matter. Either way goes, and neither is wrong.